Scenarios/Reachability/iOSReachability/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  iOSReachability
+//
+//  Created by Luis Meddrano-Zaldivar on 4/19/17.
```

Comment:
Each app needs the standard Microsoft MIT licensing notice. Check out the other samples to grab this.